lib/common/formats.js

```diff
@@ -1532,6 +1533,50 @@ declare const ${moduleName}: ${JSON.stringify(treeWalker(dictionary.tokens), nul
     });
     return flutterClassDart({ allTokens: sortedTokens, file, options, formatProperty, header });
   },
+
+  // Markdown templates
+  /**
+   * Creates a Markdown file containing a table with a row for each property.
+   *
+   * @memberof Formats
+   * @kind member
+   * @typedef {Object} markdownTableOpts
+   * @property {boolean} [markdownTableOpts.showFileHeader=true] - Whether or not to include a comment that has the build date
+   * @property {boolean} [markdownTableOpts.showDescriptionColumn=false] - Whether or not to show the description column in the table
+   * @property {OutputReferences} [markdownTableOpts.outputReferences=false] - Whether or not to keep [references](/#/formats?id=references-in-output-files) (a -> b -> c) in the output.
+   * @param {FormatArgs & { options?: markdownTableOpts }} options
+   * @example
+   * ```md
+   * | Token | Type | Value |
+   * | --- | --- | --- |
+   * | red 5 | color | `#fffaf3f2` |
+   * ```
+   */
+  'markdown/table': async function ({ dictionary, options, file }) {
+    const { allTokens, tokens, unfilteredTokens } = dictionary;
+    const { outputReferences, formatting, usesDtcg } = options;
+    const formatProperty = createPropertyFormatter({
+      outputReferences,
+      dictionary,
+      formatting,
+      usesDtcg,
+    });
```

Comment:
I think this one is redundant here, the createPropertyFormatter is mostly useful as a utility to easily put the key/value pairs into a string property matching one of these formats: css, js, sass, styles, less etc.

In the case of the markdown template, there isn't much overlap between what that does and what this util does.